specification/mediaservices/resource-manager/readme.md

```diff
@@ -62,6 +62,20 @@ directive:
     from: Accounts.json
 ```
 
+### Tag: package-metadata-2023-01
+
+These settings apply only when `--tag=package-metadata-2023-01` is specified on the command line.
+
+```yaml $(tag) == 'package-metadata-2023-01'
+input-file:
+  - Microsoft.Media/Metadata/stable/2023-01-01/AccountFilters.json
+  - Microsoft.Media/Accounts/stable/2023-01-01/Accounts.json
+  - Microsoft.Media/Metadata/stable/2023-01-01/AssetsAndAssetFilters.json
+  - Microsoft.Media/Metadata/stable/2023-01-01/ContentKeyPolicies.json
+  - Microsoft.Media/Encoding/stable/2022-07-01/Encoding.json
+  - Microsoft.Media/Metadata/stable/2023-01-01/StreamingPoliciesAndStreamingLocators.json
+  - Microsoft.Media/Streaming/stable/2022-11-01/streamingservice.json
```

Comment:
>2022-11-01 [](http://example.com/codeflow?start=37&length=10)

We might want to change this to 2022-08-01

Reply:
I will work with Sanjay and let you know.

Reply:
Sure. I see that recent change with the tag "package-account-2023-01" updated the version for streaming service.

Reply:
Reverted the api version changes and set it to 2022-11-01.